aztec-up/bin/aztec

```diff
@@ -99,6 +103,13 @@ case ${1:-} in
 
       PORTS_TO_EXPOSE="${PORTS_TO_EXPOSE:-} $P2P_TCP_LISTEN_MAPPING $P2P_UDP_LISTEN_MAPPING"
```

Comment:
Shouldn't we remove these mappings if there is a `P2P_BROADCAST_PORT` set? Feels like we should be doing either one or the other?

Reply:
yes, i was sheepish to touch this too much but i should remove it